leaf-ui/js/object/Class.js

```diff
@@ -343,7 +343,36 @@ class AnalysisResult {
         this.colorVis; //color visualization for analysis mode  
         this.selectedTimePoint; //find where slider is initialized and set timepoint in here. Also place it in update function
         this.maxTimePoint;
+
+
+        this.elements;
+        this.numOfElements;
+        this.timeScale;
+        this.relativeTime;
     }
+    // Parse results from backend
+    initFromBackEnd = function (analysisResults) {
```

Comment:
A couple things here - first up, can you rework this so that it's written in the same format as the constructor? The purpose of this rework is to move away from older JS class structures that don't look as nice! 
ex:
`initFromBackEnd() {  code here }`

I see that you call this method from displayAnalysis by passing the AnalysisResult object from the backend in. As I stated above, I'd prefer to just use the same object instead of copying everything over onto another one! Seems like a bit of a waste of time. So since this is a class method, we would not be accepting the `AnalysisResult` as an input -- Instead, if you need to reference the `AnalysisResult` object, do so using `this`.